aws/table_aws_backup_recovery_point.go

```diff
@@ -20,18 +23,14 @@ func tableAwsBackupRecoveryPoint(_ context.Context) *plugin.Table {
 		Get: &plugin.GetConfig{
 			KeyColumns: plugin.AllColumns([]string{"backup_vault_name", "recovery_point_arn"}),
 			IgnoreConfig: &plugin.IgnoreConfig{
-				ShouldIgnoreErrorFunc: isNotFoundError([]string{"NotFoundException"}),
+				ShouldIgnoreErrorFunc: isNotFoundErrorV2([]string{"NotFoundException"}),
 			},
 			Hydrate: getAwsBackupRecoveryPoint,
 		},
 		List: &plugin.ListConfig{
 			ParentHydrate: listAwsBackupVaults,
 			Hydrate:       listAwsBackupRecoveryPoints,
 			KeyColumns: []*plugin.KeyColumn{
-				{
-					Name:    "recovery_point_arn",
```

Comment:
@ParthaI why was this removed? I can see there's commented out code below when we try to use it, but does the same error in SDK v2?

Reply:
Yes, @cbruno10, the same error we are getting in SDK v2 as well. If we are passing `recovery_point_arn` value to list call param, we are getting `InvalidParameterValueException`.

Reply:
Can you please raise an issue in the AWS SDK Go v2 repo, re-add this commented out code, and add a comment below on line 190 referencing the issue? I think it's important we track it and re-enable once the AWS SDK team fixes it, or we find the correct way to use the `ByResourceArn` property.

Reply:
For the purpose of getting this PR into `main` though, along with the other AWS SDK v2 changes we have, I think OK to add this code back in for now (can comment it out though), we can merge in this PR, and then tomorrow we can add a comment referencing the issue when it's available.

Reply:
Okay, will add those back and will put a comment along with the SDK issue link.